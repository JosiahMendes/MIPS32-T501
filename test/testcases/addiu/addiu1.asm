addiu $2 $0 0x01
jr $0
addiu $2 $2 0x115
