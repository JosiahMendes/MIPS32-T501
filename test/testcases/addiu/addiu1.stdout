RAM : INIT : Loading RAM contents from test/testcases/addiu/addiu1_MEM.txt
Simulation Starting
CPU Resetting
-------------------------------------------------------------------------------------------------------------PC = bfc00000
CPU-FETCH,      Fetching instruction @ bfc00000     branch status is 0
Registers reset
                                              CPU: Register $v0 contains  00000000
CPU-DECODE      Instruction Fetched is 00000008,    reading from registers  0 and  0 
CPU-EXEC,       Register  0 (ALUInA) = 00000000,    Register  0 (ALUInB0) = 00000000,     32'Imm (ALUInB1) is 00000008      shiftamount 0
CPU-DATAMEM     Rd/Wr MemAddr(ALUOut)= xxxxxxxx,    Write data  (ALUInB0) = 00000000      Mem WriteEn =  0, ReadEn =0
CPU-WRITEBACK   Retrieved Memory     = 00000008,    Current ALUOut     =    xxxxxxxx,     Writing to Register  0...
-------------------------------------------------------------------------------------------------------------PC = bfc00004
CPU-FETCH,      Fetching instruction @ bfc00004     branch status is 2
                                              CPU: Register $v0 contains  00000000
CPU-DECODE      Instruction Fetched is 24020115,    reading from registers  0 and  2 
CPU-EXEC,       Register  0 (ALUInA) = 00000000,    Register  2 (ALUInB0) = 00000000,     32'Imm (ALUInB1) is 00000115      shiftamount 4
CPU-DATAMEM     Rd/Wr MemAddr(ALUOut)= 00000115,    Write data  (ALUInB0) = 00000000      Mem WriteEn =  0, ReadEn =0
CPU-WRITEBACK   Retrieved Memory     = 24020115,    Current ALUOut     =    00000115,     Writing to Register  2...
-------------------------------------------------------------------------------------------------------------PC = 00000000
CPU-FETCH,      Fetching instruction @ 00000000     branch status is 0
CPU HALTED
TB : Register V0 has 00000115
TB : finished; active=0
